bar_industry/demo/planning_recurrency.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="planning_recurrency_1" model="planning.recurrency">
+        <field name="last_generated_end_datetime">2025-02-17 07:00:00</field>
```

Comment:
The default values are: repeat every 1 week forever. Maybe just adapt to limit the number of repetitions:
```suggestion
        <field name="x_times">30</field>
```